src/Compat/CompatForm.php

```diff
@@ -22,4 +47,25 @@ public function hasDefaultElementDecorator()
 
         return true;
     }
+
+    /**
+     * Return a duplicate of the given submit button with the `class` attribute fixed to `primary-submit-btn-duplicate`
+     *
+     * @param FormSubmitElement $originalSubmitButton
+     *
+     * @return SubmitElement
+     */
+    public function duplicateSubmitButton(FormSubmitElement $originalSubmitButton): SubmitElement
+    {
+        $attributes = (clone $originalSubmitButton->getAttributes())
+            ->set('class', 'primary-submit-btn-duplicate');
+        $attributes->remove('id');
+
+        $submit = (new SubmitElement($originalSubmitButton->getName(), $attributes))
+            ->setValue($originalSubmitButton->getValue());
```

Comment:
You don't have to set the value here as it has no effect. The original attributes have a callback for that.